content/collections/workflow/en/experiment-learnings.md

```diff
@@ -78,10 +78,10 @@ Click the check box next to the desired notification:
 * **Experiment end reached:** You will receive this notification when your experiment is complete.
 * **SRM detected:** You will receive this notification if a [sample ratio mismatch](/docs/feature-experiment/troubleshooting/sample-ratio-mismatch) issue is identified.
 * **Long-running experiments:** You will receive this notification when your long-running experiment is complete.
-* **Statsig for the primary metric is reached:** You will receive this notification when your experiment's primary metric has reached stat sig.
+* **Statsig for the primary metric is reached:** You will receive this notification when your experiment's recommendation metric has reached stat sig.
```

Comment:
@akhil-prakash `Statsig` or `stat sig`? Can we just say statistical significance?

Reply:
either of those work. I dont know if there is any convention or not

Reply:
Cool, I recommend against `statsig` since that's [separate thing](https://amplitude.com/docs/data/destination-catalog/statsig-cohort-sync).

I'll take this on as a separate task when I'm back in the office later this week.